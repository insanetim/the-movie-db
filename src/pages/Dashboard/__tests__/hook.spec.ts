import { renderHook } from '@testing-library/react'

import useContainer from '../hook'

jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useLocation: jest.fn().mockImplementation(() => ({}))
}))

describe('Dashboard useContainer hook', () => {
  it('matches snapshot', () => {
    const { result } = renderHook(useContainer)

    expect(result.current).toMatchSnapshot()
  })
})
