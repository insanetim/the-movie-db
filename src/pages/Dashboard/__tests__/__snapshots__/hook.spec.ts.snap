// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard useContainer hook matches snapshot 1`] = `
{
  "query": "",
}
`;
